editor.cfg

```diff
@@ -0,0 +1 @@
+editorsv_enabled=true
```

Comment:
Is there any benefit to adding this to the Registry/commands.editor.setreg instead of cfg?

{
    "Amazon":
    {
        "AzCore":
        {
            "Runtime":
            {
                "ConsoleCommands":
                {
                    "editorsv_enabled": "true",
                }
            }
        }
    }
}

Reply:
discussed offline: due to precedence for using other .cfg files in this repo, going to keep it like this for now.

Reply:
What even loads editor.cfg though? I think that's a legacy load-path.. the only actual .cfg that we load at this point is I think user.cfg, and the server and client cfg files have to be explicitly loaded via command-line parameters.

Reply:
So while testing I first added this CVAR to a `user.cfg`, but decided to go with a more specifically named file to further disambiguate it's contents from the `game.cfg` (adding `editorsv_enabled` there also seems to work, but I didn't want to entangle it with whatever purpose that file was created)

If it'd be better to put this (and perhaps the contents of the `game.cfg`?) into _Registry/commands.editor.setreg_ I'm happy to do so, it just seemed like `.cfg` files were the established way to go given we had 3 of them already.

Reply:
@kberg-amzn After testing with a goofier .cfg file name, confirmed we're not arbitrarily loading any & all .cfg files from the project root (which was kind of my assumption 😬). 

It looks like [`editor.cfg`](https://github.com/o3de/o3de/blob/development/Code/Editor/CryEdit.cpp#L1283) (and [`game.cfg`](https://github.com/o3de/o3de/blob/development/Code/Legacy/CrySystem/SystemInit.cpp#L341)!) are being loaded by some old _Cry_ code, whose original purpose I'm reasonably sure didn't have to do with arbitrary local server cvars soooooo I am going to submit a follow up PR to move this into a .setreg file, as suggested by @AMZN-Gene 

Thanks for the tip that .cfg file values shouldn't just be loading from thin air!